typescript-carousel/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TypeScript Carousel</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="layout.css" />
+  </head>
+  <body>
+    <div class="row align-center justify-space-evenly carousel-container">
+      <i class="fa-solid fa-chevron-left fa-2xl"></i>
+      <div class="column justify-center">
+        <img src="" class="display-image" />
+        <div class="row justify-center">
+          <i class="circ-0 fa-solid fa-circle"></i>
+          <i class="circ-1 fa-regular fa-circle"></i>
+          <i class="circ-2 fa-regular fa-circle"></i>
+          <i class="circ-3 fa-regular fa-circle circ-3"></i>
+          <i class="circ-4 fa-regular fa-circle"></i>
```

Comment:
I would have just given these an `id` attribute or a `data-index` attribute and used that value instead of trying to take it from the class.